### DISCUSSION TITLE
Filter rule `equals variable`

### DISCUSSION THREAD

**Discussion by JLKM** [directus/directus#10751] [Feature Requests]
Being able to filter the pool of subcategories based on their (main)categories is a much needed feature in my use case - and likely in many other use cases as well.

The filter rules on M2O-fields in V9 are real powerful in many respects, but it looks like, you are only able to filter the output based on fixed/constant filter settings like existing keywords from a dropdown or a manual input.

A filter rule based on the variable value from a given field on the actual item or on related items - would be a big leep forward.

To put it more programmatically:
- `if this_item.field1.value equals/contains lookup_item.field1.value then return lookup_item.field2.values as a unified choice list in this_item.field2`.

Further info can be found in [#9135](https://github.com/directus/directus/pull/9135)

**Comment by cdiaz** [comment]
Apparently [this ](https://github.com/directus/directus/discussions/10612) is same

**Reply by rijkvanzanten** [reply]
> I didn't get how to check for null field using _null?

While technically unrelated to this discussion, the answer is:

```json
{
  "field": {
    "_null": true
  }
}
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
